docs/usage/networking.md

```diff
@@ -0,0 +1,8 @@
+# Networking
```

Comment:
Do we really be so granular? One file for each of these things? Without proper linking structure this becomes a bit hard to navigate.

Why not a some sort of FAQ page where we can aggregate at least some of the "smaller" things